com.github.textosaurus.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "com.github.textosaurus",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "textosaurus",
+    "rename-icon": "textosaurus",
+    "finish-args": [ "--device=dri",
+                     "--share=ipc",
+                     "--share=network",
+                     "--socket=x11",
+                     "--socket=wayland",
+                     "--filesystem=xdg-download",
```

Comment:
Surely as a text editor you want more access than that.

Reply:
Sure.